charts/astronomer/templates/houston/worker/houston-worker-deployment.yaml

```diff
@@ -94,7 +94,7 @@ spec:
           command:
             {{- toYaml .Values.houston.worker.command | nindent 12 }}
           {{ else }}
-          command: ["yarn", "worker"]
+          command: ["/bin/sh", "-c", "update-ca-certificates;yarn worker"]
```

Comment:
@danielhoherd @rishkarajgi should this be added behind the flag or lets run them as bydefault

Reply:
@danielhoherd

Reply:
@pgvishnuram if we run this by default we need to think about how it can fail and what will happen when it does fail.

Reply:
since we have fixed the perm issue at max it wont fail and also houston always runs this command by default via entrypoint

Reply:
or i can add this when global.privatecaenabled

Reply:
i'm fine with us running this every time as long as the pod will not fail if this command exits non-zero.